src/server_manager/ui_components/outline-server-view.html

```diff
@@ -701,9 +702,10 @@ <h3>[[localize('data-limits-dialog-title')]]</h3>
       addAccessKey: function(accessKey) {
         // TODO(fortuna): Restore loading animation.
         // TODO(fortuna): Restore highlighting.
-        // Push to array using Polymer in order to propagate changes. See
-        // https://www.polymer-project.org/1.0/docs/devguide/model-data#array-mutation
         this.push("accessKeyRows", accessKey);
+        this.$.accessKeysContainer.querySelector("dom-repeat").render();
```

Comment:
Add a comment explaining you need to render first in order to get the input element.

Reply:
Done